lib/facter/java_version.rb

```diff
@@ -11,6 +11,11 @@
 #
 # Notes:
 #   None
+osfamily = Facter.value(:osfamily)
+case osfamily
+when 'OpenBSD'
+  ENV['PATH'] = ENV['PATH'] + File::PATH_SEPARATOR + '/usr/local/jdk-1.7.0/jre/bin'
```

Comment:
Could this be changed so it works with any java version? So just add in `/usr/local/` to the path or something?